src/datasets/index.ts

```diff
@@ -14,7 +14,7 @@ export const ECOSYSTEM: EcosystemElement[] = [...ECOSYSTEM_SDKS, ...ECOSYSTEM_PR
 export const TECHNOLOGY_COLOR_MAP: Record<Technology, string> = {
   JavaScript: 'bg-yellow-50 text-yellow-600 ring-yellow-500/10',
   Go: 'bg-sky-50 text-sky-600 ring-sky-500/10',
-  Java: 'bg-amber-50 text-amber-600 ring-amber-500/10',
+  Java: 'bg-rose-50 text-rose-600 ring-rose-500/10',
```

Comment:
Was there a reason to change this? I tried to match these colors with what GitHub uses.

Reply:
was trying to reduce the overlap with the colours for SvelteKit, Java seems to use more red as their branding. But happy to try and match the Github ones, where did you find them?